src/sbf.cpp

```diff
@@ -78,17 +78,29 @@ GPSDriverSBF::~GPSDriverSBF()
 
 int GPSDriverSBF::configure(unsigned &baudrate, const GPSConfig &config)
 {
+	char buf[GPS_READ_BUFFER_SIZE];
+	char msg[MSG_SIZE];
+
 	_configured = false;
 
 	setBaudrate(SBF_TX_CFG_PRT_BAUDRATE);
 	baudrate = SBF_TX_CFG_PRT_BAUDRATE;
 	_output_mode = config.output_mode;
 
+	// Make sure we can send commands to the receiver
 	sendMessage(SBF_CONFIG_FORCE_INPUT);
 
-	char buf[GPS_READ_BUFFER_SIZE];
-	char com_port[5] {};
+	// Disable previous output for now so we can detect the COM port
+	for (int i = 1; i <= 2; i++) {
```

Comment:
This had me very confused. I'm used to:
```suggestion
	for (int i = 0; i < 2; i++) {
```

Reply:
I always use the former for indices and the latter for real values (e.g. COM1 to COM2). I can change it if this is confusing.

Reply:
I see. I guess it's fine.